### PR TITLE
NEFI 81 crud customers update

### PR DESCRIPTION
**Pull request by vilapedro92**
### **Description**
This PR adds the basic features for update a customers

### **Main changes**
Added the route to update a customer
The controller was added to update a customer
Added the service to update a customer
The model was added to update a customer

### **Side notes**

Type of request => PUT
Endpoint => http://localhost:3001/api/v1/customer/3a404443-db83-476c-9aed-f31af10e78a3
Parameters => id 
Body => { firstname: "MyFirstName", lastname: "MyLastName" }
Aditional Information => Max 32 characters for Firstname and Lastname. 

### **Response:**
Return the customer updated

{
    "id": "3a404443-db83-476c-9aed-f31af10e78a3",
    "user_id": "4d297634-7388-4a9a-8a07-46ff14dc5cbd",
    "external_id": null,
    "display_name": "Pedro Villa",
    "display": "PedroVilla",
    "customer_contact_field": {
        "customer_id": "3a404443-db83-476c-9aed-f31af10e78a3",
        "contact_field_id": "TESTING",
        "value": "+573001110001"
    },
    "conversations": [
        {
            "attendee_id": "3a404443-db83-476c-9aed-f31af10e78a3",
            "recipient_id": "testing-+13330004444",
            "host_id": null
        }
    ],
    "customer_fields": {}
}

### CODE REVIEW

File: api/src/v1/services/user.service.js [hunk 1]
```diff
@@ -24,10 +26,14 @@ class UserServiceClass extends BaseModelService {
...
 
-  update(user) {
-    const {firstname, lastname, alias, id} = user;
-    const payload = {firstname, lastname, alias, id};
-    return this.model.update(payload);
+  async update(user) {
+    const {firstname, lastname, id} = user;
+    const payload = {firstname, lastname};
+    const {User} = shemas;
+    if (!(await User.isValid(payload))) {
```
Comment (alphonse92) [inline api/src/v1/services/user.service.js:33]: As far I know, `isValid` function throws an exception, so the if is never reached. Please remove the if validation, then, wrap it inside a try catch statement, then if the validation throws an error then throws `throw new BadParameters('Firstname and lastname are Invalid');`

**Review by alphonse92 — CHANGES_REQUESTED** [review]

File: api/src/v1/services/user.service.js (hunk 1 above)
Comment (ctabares-uruit) [reply api/src/v1/services/user.service.js:34]: imagen
Validation actually works with true of false return, no with a thrown error.

Comment (alphonse92) [reply api/src/v1/services/user.service.js:34]: ty for checking this out

**Review by alphonse92 — APPROVED** [review]
Thank you so much

**Pull request merged**